library/alloc/src/gc.rs

```diff
@@ -257,6 +257,26 @@ impl<T> Gc<T> {
 }
 
 impl Gc<dyn Any> {
+    /// Attempt to downcast the `Gc<dyn Any>` to a concrete type.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # #![feature(gc)]
```

Comment:
Is the `# #!` bit deliberate (i.e. two `#`s)?

Reply:
Yes, the first `#` means that the line is not rendered in the docs example but it's needed for the example to compile. It's a way to elide setup code that make the examples harder to read.

Reply:
Aha!